.github/workflows/code_style.yml

```diff
@@ -28,13 +28,15 @@ jobs:
           cmake -DENABLE_PYTHON=ON -DENABLE_TESTS=ON -DENABLE_PROFILING_ITT=ON -DSELECTIVE_BUILD=COLLECT ..
 
       - name: Check code style
-        run: cmake --build build --target clang_format_check_all -j8
+        run: cmake --build . --target clang_format_check_all -j8
+        working-directory: build
 
       - name: Create code style diff
         if: failure()
         run: |
-          cmake --build build --target clang_format_fix_all
+          cmake --build . --target clang_format_fix_all
```

Comment:
We can also use `-j8`

Reply:
It's run only in case of failure, I expected we don't have a lot of failures to run them from parallel threads :)